docs/guide/snaps.md

```diff
@@ -38,36 +38,36 @@ There are a number of simple snaps available for you to learn from and begin to
+[MetaMask Flask](https://metamask.io/flask) is a distribution channel for developmental, experimental, and kooky stuff for MetaMask. Snaps is just the first feature to be released through it.
 
-- Start by downloading [Flask](https://metamask.io/flask) and adding it to the available browser of your choice
+- Start by downloading [Flask](https://metamask.io/flask) and adding it to one of the available browsers of choice
 
-At this point, you should see Flask added to your browser as an extension. If you currently have another build of MetaMask installed in your browser, disable it (but make sure you have your Secret Recovery Phrase backed up!).
+After adding Flask as a browser extension, if you currently have another build of MetaMask installed in your browser, disable it (ensure you have your Secret Recovery Phrase backed up!).
 
-### Serving a snap to your local environment
+### Serving a Snap to Your Local Environment
 
-Now that you have Flask in your browser, it's time to find something to plug into it. If you don't want to download any code for the moment, check out the example snap [deployed here](https://metamask.github.io/snap-template/) and follow the instructions below under 'Interacting with the Snap'.
+With Flask in your browser, it's time to plug something into it. If you don't want to download any code for the moment, check out the example snap [deployed here](https://metamask.github.io/snap-template/) and follow the instructions below under 'Interacting with the Snap'.
 
 If you're ready to jump in and see how a Snap looks on the backend, follow the steps below!
 
 Open up a new terminal window, and do the following:
 
 - clone the snaps monorepo [here](https://github.com/MetaMask/snaps-skunkworks)
-- `cd` into the repo, and run `nvm use 14.15` to make sure you're using the right version of nodejs
+- `cd` into the repo, and run `nvm use 14.15` our reccomended version of nodejs
 - navigate to the "Hello, World!" snap: `cd packages/example-snap`
 - install [@metamask/snaps-cli](https://npmjs.com/package/@metamask/snaps-cli) package locally: `yarn add snaps-cli`
 - run `yarn mm-snap serve`
 
-Your terminal window should now be serving the example Snap locally. Time to see the functionality in action!
+Your terminal window should now be serving the example snap locally. Time to see the functionality in action!
 
-### Interacting with the snap
+### Interacting with the Snap
 
 If you're serving the snap locally, open up a browser window, and navigate to `http://localhost:8080`.
 
-You should see a simple message, explaining the functionality of the snap, and two buttons. Start by clicking the `Connect` button; your Flask extension should pop up, and require you to click through two notifications.
+You should see a simple message explaining the functionality of the snap, and two buttons. Start by clicking the `Connect` button; your Flask extension should pop up and require you to click through two notifications.
```

Comment:
Do we need the comma in the first sentence?

Reply:
I think it helps clarify that the buttons are listed as things the user should see, rather than as being explained by the message.

Reply:
Ah, that's right.